nft_ingester/src/backfiller.rs

```diff
@@ -182,7 +182,7 @@ where
         it.seek_to_first();
     }
     while it.valid() {
-        if shutdown_token.is_cancelled() {
+        if cancellation_token.is_cancelled() {
```

Comment:
The same concern. It's not a problem in general if the rest of the body will take a short time to complete because then we will guarantee that a message is processed and not lost though.

Reply:
each iteration here seems to be complete, we don't batch it or whatever; i think what we had before is fine